benchmark/Streamly/Benchmark/Prelude/Serial/Nested.hs

```diff
@@ -160,12 +160,12 @@ inspect $ 'mergeByM `hasNoType` ''D.Step
 o_1_space_joining :: Int -> [Benchmark]
 o_1_space_joining value =
     [ bgroup "joining"
-        [ benchIOSrc1 "serial (2,x/2)" (serial2 (value `div` 2))
-        , benchIOSrc1 "append (2,x/2)" (append2 (value `div` 2))
-        , benchIOSrc1 "serial (2,2,x/4)" (serial4 (value `div` 4))
-        , benchIOSrc1 "append (2,2,x/4)" (append4 (value `div` 4))
-        , benchIOSrc1 "mergeBy (2,x/2)" (mergeBy (value `div` 2))
-        , benchIOSrc1 "mergeByM (2,x/2)" (mergeByM (value `div` 2))
+        [ benchIOSrc1 "serial (2 - x/2)" (serial2 (value `div` 2))
+        , benchIOSrc1 "append (2 - x/2)" (append2 (value `div` 2))
+        , benchIOSrc1 "serial (2 - 2 - x/4)" (serial4 (value `div` 4))
+        , benchIOSrc1 "append (2 - 2 - x/4)" (append4 (value `div` 4))
+        , benchIOSrc1 "mergeBy (2 - x/2)" (mergeBy (value `div` 2))
+        , benchIOSrc1 "mergeByM (2 - x/2)" (mergeByM (value `div` 2))
```

Comment:
Why was this changed?

Reply:
comma "," is causing parsing issue tasty-bench.

Reply:
We need to fix the script, if possible, rather than changing the benchmark names.

Reply:
Benchmark names with "," are printed in double quotes e.g.

```
"All.Data.Fold/o-1-space.composition.tee (all, any)",75139802,5497826
```